dbms_test.go

```diff
@@ -0,0 +1,163 @@
+package sqllexer
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
```

Comment:
It might be worth considering [embed](https://pkg.go.dev/embed) for this instead of direct file/directory apis. I don't know for sure this would be a problem here but sometimes we run into issues with the file paths being invalid/wrong on different environments. Embed basically maps a filesystem on top of a directory/files relative to the code that defines it so it's more bulletproof that way. 

And it still offers you the same APIs for reading directories and loading files.

Reply:
Good suggestion. Updated with `ebmed`